### ISSUE TITLE
REST api choice

### ISSUE THREAD

**Issue by mbnoimi** [cutelyst/cutelyst#268]
Hi,

After reading some issues related to REST I just want to be sure; 
Does Cutelyst is a good choice for building REST web services (just like [DRF](https://www.django-rest-framework.org/))?

**Comment by dantti** [comment]
Well I don't know what your requirements are, but for sure you will use way less memory and it will be a lot faster.
I use it on production for many projects from big to small I find it a clear win.

**Comment by mbnoimi** [comment]
> Well I don't know what your requirements are, but for sure you will use way less memory and it will be a lot faster.
> I use it on production for many projects from big to small I find it a clear win.

Are you familiar with DFR?

**Comment by dantti** [comment]
No, but looking at it's web site it's built upon Django which is slower and uses a lot more memory.

**Comment by mbnoimi** [comment]
> No, but looking at it's web site it's built upon Django which is slower and uses a lot more memory.

Actually I want to balance my choice between rapid developing and performance. Cutelyst is much faster depending on the mentioned benchmark but I afraid of the small community. I can see the most replies come from you while most of the community is just like me (newcomers) so this issue really concerns me.
I could successfully build my first project after the nightmare of building Cutelyst and I feel in joy right now but I don't want to rash into without studying the consequences.

My requirements are:
1. Framework uses C++/Qt
2. Medium performance.
3. Easy to build REST web services (asynchronous, serialization, routing, authorization, openapi compatible...etc )
4. ORM
5. Embedded HTTP server

Most of them are already available in Cutelyst which is really cool but the community issue still nagging me.

**Comment by mbnoimi** [comment]
BTW, can I use `Cutelyst-WSG` in production (I usually use Apache but I prefer to use `Cutelyst-WSG` because of its performance )?

**Comment by dantti** [comment]
Well there are at least 3 more active users/developers, sadly Qt/C++ specially in Web Dev isn't the hype or trend, but that doesn't mean it isn't a good choice, the code is LGPL the same open source license of Qt, and if you know Qt you can also read the code and propose fixes.

For the cutelyst-wsgi yes you can use it on production, the other alternative is uWSGI which isn't recommended anymore.
I myself use nginx + cutelyst-wsgi because I need virtual hosts and serving files with nginx is better.

**Comment by mbnoimi** [comment]
>  I myself use nginx + cutelyst-wsgi because I need virtual hosts and serving files with nginx is better.

May you please add a new wiki page guide us how to deploy Cutelyst web app to nginx?
